src/generators/minecraft-character/MinecraftCharacterGenerator.res

```diff
@@ -170,24 +100,80 @@ let script = () => {
     true,
   )
 
-  Generator.defineRegionInput((72, 24, 264, 198), () => {
-    Generator.setBooleanInputValue("Show Head Overlay", !showHeadOverlay)
-  })
-  Generator.defineRegionInput((266, 198, 198, 166), () => {
-    Generator.setBooleanInputValue("Show Body Overlay", !showBodyOverlay)
-  })
-  Generator.defineRegionInput((382, 372, 134, 166), () => {
-    Generator.setBooleanInputValue("Show Left Arm Overlay", !showLeftArmOverlay)
-  })
-  Generator.defineRegionInput((99, 372, 134, 166), () => {
-    Generator.setBooleanInputValue("Show Right Arm Overlay", !showRightArmOverlay)
-  })
-  Generator.defineRegionInput((383, 584, 134, 166), () => {
-    Generator.setBooleanInputValue("Show Left Leg Overlay", !showLeftLegOverlay)
-  })
-  Generator.defineRegionInput((99, 584, 134, 166), () => {
-    Generator.setBooleanInputValue("Show Right Leg Overlay", !showRightLegOverlay)
-  })
+  let char = isAlexModel ? alex : steve
+
+  let drawHead = ((ox, oy): Generator_Builder.position) => {
+    let scale = (64, 64, 64)
+    Minecraft.drawCuboid("Skin", char.base.head, (ox, oy), scale, ())
```

Comment:
Minor: You might like to replace `((ox, oy): Generator_Builder.position)` above with `(position: Generator_Builder.position)`, and then just use `position` everywhere?

Or, alternatively, just use `(ox: int, oy: int)`

Or, feel free to keep as is - whatever you prefer is fine.